travis/shared_configs/python-tester-conda-latest.yml

```diff
@@ -54,3 +55,9 @@ jobs:
           else
             echo "Logfile ${LOGFILE} not found"
           fi
+    - <<: *testpythonconda
+      name: "Python 3.10"
+      env:
+        - PYTHON_VERSION: 3.10
```

Comment:
I seem to recall that there was some scenario where yaml interpretation of floating point values caused issues when interpreting Python version numbers. 

Seems like this worked for you, though, so perhaps my memory is just faulty and quotes won't be required going forward.

Reply:
oh shoot I forgot about this, I'll at least look into it a bit

Reply:
It certainly worked here but I want to be super careful if it ever stops working. The travis docs have a mix of quoted and non-quoted. I'll adjust.

Reply:
This is an environment variable that gets gobbled into a python command so it should continue to work

Reply:
Gotcha - I think we both agree on quoting being safer/more explicit, at least, so 👍 on ed34071 from me

Reply:
The lucid CI seemed to load py3.10, but adding quotes seems worthwhile.

Reply:
I'm going to exercise the CI test on the lucid PR once more just in case the quote broke something (paranoia)